overlay-manager.html

```diff
@@ -0,0 +1,108 @@
+<!--
+    @license
+    Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+    This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+    The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+    The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+    Code distributed by Google as part of the polymer project is also
+    subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+
+<script>
+
+  Polymer.Core = Polymer.Core || {};
+  Polymer.Core.Overlay = Polymer.Core.Overlay || {};
```

Comment:
This ad hoc namespace management is worrisome, but I'm not sure I have a better suggestion.